pkg/sql/colexec2/connector/connector_test.go

```diff
@@ -123,6 +123,7 @@ func newTestCase(gm *guest.Mmu) connectorTestCase {
 // create a new block based on the type information
 func newBatch(t *testing.T, ts []types.Type, proc *process.Process, rows int64) *batch.Batch {
 	bat := batch.New(len(ts))
+	bat.Cnt = 1
```

Comment:
Atomic needed in order to avoid race condition.

Reply:
This does not result in a race, just as new(sync.Mutex) does. The lock instruction is not required for initialization.

Reply:
Yes, you're right. I didn't check its context.